tests/integration/api_issue_pin_test.go

```diff
@@ -148,7 +148,7 @@ func TestAPIListPinnedIssues(t *testing.T) {
 	assert.NoError(t, unittest.LoadFixtures())
 
 	repo := unittest.AssertExistsAndLoadBean(t, &repo_model.Repository{ID: 1})
-	issue := unittest.AssertExistsAndLoadBean(t, &issues_model.Issue{RepoID: repo.ID})
+	issue := unittest.AssertExistsAndLoadBean(t, &issues_model.Issue{RepoID: repo.ID, IsPull: false})
```

Comment:
`IsPull: false` doesn't work, because `false` is a zero value, it is omitted by the XORM framework IIRC.